example-react-app/src/app/associationM2M-management/AssociationM2MBrowserTable.tsx

```diff
@@ -149,7 +151,7 @@ const AssociationM2MBrowserTable = observer(
         error={error}
         enableFiltersOnColumns={entityList != null ? [] : undefined}
         enableSortingOnColumns={entityList != null ? [] : undefined}
-        columnDefinitions={["name"]}
+        columnDefinitions={COLUMN_NAMES}
```

Comment:
Our current agreed approach is to keep column definitions here instead of putting them into constant, as it is more readable.